modules/core/shared/src/main/scala/lucuma/core/util/syntax/package.scala

```diff
@@ -3,6 +3,11 @@
 
 package lucuma.core.util
```

Comment:
Suggest moving this up to `lucuma.core.syntax` so we don't have to dig for it. This ended up being a problem in OCS.

Reply:
Makes sense.

Reply:
Do we want to do `import lucuma.core.syntax._` or `import lucuma.core.syntax.all._`

Reply:
I think `import lucuma.core.syntax.all._` … I think should do what cats does in these cases.

Reply:
There is currently a lot of syntax under `lucuma.core.math.syntax`. Should we move that, too?

Reply:
Hm. I was initially thinking that everything in `gsp.math` would move into `lucuma.core` rather than `lucuma.core.math` but I guess we need to discuss it more. It's nice to have all the instances and syntax in one place.

Reply:
We can discuss in slack, and I'll move the `lucuma.core.math` syntax to `lucuma.core` in another PR if we decide to do so.